components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/AuthorizationGrantCacheEntry.java

```diff
@@ -258,4 +262,24 @@ public void setSessionContextIdentifier(String sessionContextIdentifier) {
 
         this.sessionContextIdentifier = sessionContextIdentifier;
     }
+
+    public void setOidcSessionId(String oidcSessionId) {
```

Comment:
can we add a comment here saying it is used as for sid claim. People might be confused with session context identifier

Reply:
Added with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1617/commits/4929f4b953297e502901f0dacc1a6fab523ded95